akka-cluster-sharding/src/test/scala/akka/cluster/sharding/internal/RememberEntitiesShardStoreSpec.scala

```diff
@@ -17,17 +19,21 @@ import org.scalatest.wordspec.AnyWordSpecLike
  * Covers the interaction between the shard and the remember entities store
  */
 object RememberEntitiesShardStoreSpec {
-  def config = ConfigFactory.parseString("""
+  def config = ConfigFactory.parseString(s"""
       akka.loglevel=DEBUG
       akka.loggers = ["akka.testkit.SilenceAllTestEventListener"]
       akka.actor.provider = cluster
       akka.remote.artery.canonical.port = 0
       akka.remote.classic.netty.tcp.port = 0
       akka.cluster.sharding.state-store-mode = ddata
+      akka.cluster.sharding.snapshot-after = 2
```

Comment:
I think this, and config for a snapshot store below is the only thing that needs to be changed in the test to exercise the bug and that you can skip the added test case.

Reply:
Thanks. I thought the additional test might be easier to read but it does not seem to work with DDataRememberEntitiesShardStore. I removed the test and an unrelated config change.